src/Microsoft.Azure.ServiceBus/IQueueClient.cs

```diff
@@ -14,5 +18,27 @@ public interface IQueueClient : IReceiverClient, ISenderClient
         /// Gets the name of the queue.
         /// </summary>
         string QueueName { get; }
+
+        /// <summary>
+        /// Receive session messages continously from the queue. Registers a message handler and begins a new thread to receive session-messages.
```

Comment:
actually its fine

---
In reply to: [124972167](https://github.com/Azure/azure-service-bus-dotnet/pull/202#discussion_r124972167) [](ancestors = 124972167)